qube/create/templates/portlet/portlet_groovy_osgi/cookiecutter.json

```diff
@@ -0,0 +1,14 @@
+{
+    "full_name": "Homer Simpson",
+    "email": "homer.simpson@qbic.uni-tuebingen.de",
+    "project_name": "sample-cli",
+    "project_slug": "sample-cli",
+    "main_class_prefix": "Sample",
+    "version": "0.1.0-SNAPSHOT",
+    "license": "MIT",
```

Comment:
I copied this file. Is this then a bug for all templates? If yes I would suggest to add the license key to all templates and add this to the current issue

Reply:
I just created a library from the Java lib template, which has the license property missing, and it works there. So this is a good hint to start the investigation, maybe we introduced a bug.

Reply:
Maybe this was always a bug since the license itself was never used before in the README (see my comment for the usage of the variable that I added). Still, I am not sure if it was used on another place